src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -578,9 +590,9 @@ public void cannotCreateRequestWithUserBelongingToNoPatronGroup() {
       .by(noUserGroupBob));
 
     assertThat(recallResponse, hasStatus(HTTP_UNPROCESSABLE_ENTITY));
-
-    assertThat(recallResponse.getJson(), hasErrorWith(allOf(
-      hasMessage("A valid patron group is required. PatronGroup ID is null"))));
+    assertThat(recallResponse.getJson(), hasErrors(1));
+    assertThat(recallResponse.getJson(), hasErrorWith(
+      hasMessage("A valid patron group is required. PatronGroup ID is null.")));
```

Comment:
Why has the error message changed?

Reply:
It was changed since I split validation of user and patron group ID into two separate validations. But this separation was no longer necessary (see Martin's comment above) so I reverted it. This error message is now back to it's original state.

Reply:
Have the other error messages that had changed also been reverted?

Reply:
Yes. Two error messages were changed as a result of this split, and both were reverted.